imapserver/search.go

```diff
@@ -98,15 +115,15 @@ func (c *Conn) writeESearch(tag string, data *imap.SearchData, options *imap.Sea
 
 	enc.Atom("*").SP().Atom("ESEARCH")
 	if tag != "" {
-		enc.SP().Special('(').Atom("TAG").SP().Atom(tag).Special(')')
+		enc.SP().Special('(').Atom("TAG").SP().String(tag).Special(')')
```

Comment:
Good catch. In IMAP4rev2 this is an `astring`, but in RFC 4466 this must be a `string`.